docker-compose.yml

```diff
@@ -1,37 +1,35 @@
 version: '2'
 
 volumes:
-    db-data:
-    vendor:
-    var:
+  db-data:
+  vendor:
+  var:
 
 services:
-    mysql:
-        image: mysql:5
-        ports:
-            - "3306"
-        volumes:
-            - db-data:/var/lib/mysql
-        environment:
-            MYSQL_ROOT_PASSWORD: password
-            MYSQL_DATABASE: prestashop
-        restart: always
-    apache:
-        container_name: prestashop-git
-        image: prestashop/prestashop-git:7
-        environment:
-            PS_INSTALL_AUTO: 1
-            DB_PASSWD: password
-            DB_NAME: prestashop
-            DB_SERVER: mysql
-            PS_DOMAIN: localhost:8001
-            PS_FOLDER_INSTALL: install-dev
-            PS_FOLDER_ADMIN: admin-dev
-        ports:
-            - "8001:80"
-        volumes:
-            - ./:/var/www/html/:delegated
-            - vendor:/var/www/html/vendor
-            - var:/var/www/html/var
-        restart: always
-
+  mysql:
+    image: mysql:5
+    ports:
+      - "3306"
+    volumes:
+      - db-data:/var/lib/mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: prestashop
+      MYSQL_DATABASE: prestashop
+    restart: always
+  prestashop-git:
+    build: .docker
+    environment:
+      PS_INSTALL_AUTO: 1
+      DB_PASSWD: prestashop
+      DB_NAME: prestashop
+      DB_SERVER: mysql
+      PS_DOMAIN: localhost:8001
+      PS_FOLDER_INSTALL: install-dev
+      PS_FOLDER_ADMIN: admin-dev
+    command: ["/tmp/wait-for-it.sh", "--timeout=60", "--strict", "mysql:3306", "--", "/tmp/docker_run_git.sh"]
```

Comment:
So you need to run this command in both `docker-compose.yml` and in `.docker/Dockerfile` ?

Reply:
Dockerfile is not related to the docker-compose behavior.
In `.docker/Dockerfile` we are building the container architecture / behavior, whereas when you use the docker-compose, you are waiting for some service to be up